File: [1] BEGINNER/1065 - Pares entre Cinco Números.py
'''
Faça um programa que leia 5 valores inteiros. Conte quantos destes valores digitados são pares e mostre esta informação.

Entrada
O arquivo de entrada contém 5 valores inteiros quaisquer.

Saída
Imprima a mensagem conforme o exemplo fornecido, indicando a quantidade de valores pares lidos.
'''

valores = []
pares = 0
for i in range(5):
    num = int(input())
    valores.append(num)
for i in valores:
    if i % 2 == 0:
        pares += 1
print('{} valores pares'.format(pares))

File: [1] BEGINNER/1065 - Pares entre Cinco Números.hs
{-
Faça um programa que leia 5 valores inteiros. Conte quantos destes valores digitados são pares e mostre esta informação.

Entrada
O arquivo de entrada contém 5 valores inteiros quaisquer.

Saída
Imprima a mensagem conforme o exemplo fornecido, indicando a quantidade de valores pares lidos.
-}

import Text.Printf

main = do
a <- readLn
b <- readLn
c <- readLn
d <- readLn
e <- readLn
let lista = [i | i <- [a,b,c,d,e], rem i 2 == 0]
let tamanho = length (lista)

printf "%d valores pares\n" tamanho